contracts/queue/src/contract.rs

```diff
@@ -0,0 +1,202 @@
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+use snafu::ResultExt;
+
+use cosmwasm_std::{
+    from_slice, to_vec, Api, Binary, Env, Extern, Order, ParseErr, Response, Result, SerializeErr,
+    Storage,
+};
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+pub struct InitMsg {}
+
+// we store one entry for each item in the queue
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+pub struct State {
+    pub value: i32,
+}
+
+// failure modes to help test wasmd, based on this comment
+// https://github.com/cosmwasm/wasmd/issues/8#issuecomment-576146751
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[serde(rename_all = "lowercase")]
+pub enum HandleMsg {
+    // Push will add some value to the end of list
+    Push { value: i32 },
+    // Pop will remove value from start of the list
+    Pop {},
```

Comment:
"push" and "pop" is the typical naming of a stack (name coming from those devices https://gastro-technik.net/Fahrbare-Geraete-Edelstahlwagen/Hupfer-Tablettstapler/Hupfer-Edelstahl-Tablettstapler-SGR-OTA-53-37-S::9077.html):
- https://en.wikipedia.org/wiki/Stack_(abstract_data_type)
- https://en.wikipedia.org/wiki/Queue_(abstract_data_type)

I'd renaming the operations to "enqueue" and "dequeue".